docs/source/user_guide/connector/config.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading configuration files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When ```DataPrep.Connector.connect()``` is called, the system reads from a configuration folder, which contains information about how to build connection with specific APIs.\n",
+    "\n",
+    "You can load the configuration files in the following two ways.\n",
+    "\n",
+    "### 1. Loading existing files from our github repo\n",
+    "\n",
+    "We maintain a github repo that contains configuration files for more than 20 websites.\n",
+    "\n",
+    "//todo: add reference\n",
+    "\n",
+    "As an example, with the following code, the system will first download the configuration file folder of DBLP from our repo, and then load it to build the connection.\n",
+    "The file is placed at the system temporary file folder.\n",
+    "\n",
+    "```\n",
+    "from dataprep.connector import connect\n",
+    "conn = connect(\"dblp\")\n",
+    "```\n",
+    "\n",
+    "```connect()``` provides a parameter called ```update```, which forces update the config files if set to ```True```.\n",
+    "\n",
+    "\n",
+    "### 2. Loading from a local directory\n",
+    "\n",
+    "You can also load your own configuration files from a local directory. \n",
+    "For example, by writing the following code, you should have a folder called ```dblp``` in parallel with your code file. Connector will load from the corresponding folder and build the connection.\n",
```

Comment:
The wording of this is a bit confusing.

Reply:
Changed it a bit. Let me know if it makes sense.